apps/src/templates/progress/SendLessonDialog.jsx

```diff
@@ -53,24 +66,15 @@ export default class SendLessonDialog extends Component {
           style={styles.button}
           onClick={() => copyToClipboard(this.props.lessonUrl)}
         />
-        {i18n.sendLessonCopyLink()}
+        <span style={styles.buttonLabel}>{i18n.sendLessonCopyLink()}</span>
       </div>
     );
   }
 
   renderShareToGoogleRow() {
     return (
-      <div>
-        {/* TODO: Replace placeholder button with actual implementation */}
-        <Button
-          text=""
-          icon="users"
-          iconStyle={styles.buttonIcon}
-          color="white"
-          style={styles.button}
-          onClick={() => {}}
-        />
-        {i18n.sendLessonShareToGoogle()}
+      <div style={styles.row}>
+        <GoogleClassroomShareButton height={48} url={this.props.lessonUrl} />
```

Comment:
oh wait, one thing: when we're opening this dialog for a google section, i think we should pass the courseid into the button so it pre-selects the corresponding google classroom. can you add that please?

Reply:
Thanks for noticing this!  I was going to plumb through the rest of the google classroom props (courseid, title, and analyticsData) when I hook up the dialog to the unit overview page.  Would that be ok?

Reply:
sounds good!